src/shims/unix/linux/epoll.rs

```diff
@@ -39,6 +37,7 @@ impl FileDescription for Epoll {
     fn close<'tcx>(
         self: Box<Self>,
         _communicate_allowed: bool,
+        _ecx: &mut MiriInterpCx<'tcx>,
```

Comment:
I'd prefer to just leave it out and add it in the other PR. After removing the integer ids from this PR there should not be too many conflicts